resources/metadata/cnbBuild.yaml

```diff
@@ -363,3 +363,6 @@ spec:
             type: sbom
   containers:
     - image: "paketobuildpacks/builder:base"
+      options:
+        - name: -u
+          value: "0"
```

Comment:
Does it force to run the image under root?

Reply:
it enforces to run piper as root, the user for CNB lifecycle will be taken from the `CNB_USER_ID` and `CNB_GROUP_ID`. When the process is invoked, privileges are dropped down to this gid/uid